bin/pycbc_live

```diff
@@ -291,7 +291,7 @@ class LiveEventManager(object):
                     hdfp['template_duration'] = \
                         bank.table[template_id].template_duration
                     hdfp['ifar'] = ifar
-                    hdfp['gid'] = gid
```

Comment:
I do not think this is necessary. If `gid` is `None`, so be it. Or does it create problems downstream?

Reply:
Actually, even better:
```python
if gid is not None:
    hdfp['gid'] = gid
```

Reply:
None read from hdf did create problem. Hence the case.

Reply:
I see. Ideally `gid` should be an attribute instead of a dataset, I think, then `None` should be ok. Would you mind implementing my suggestion anyway?

Reply:
Ok. I will try that. Else I will convert it to attr.

Reply:
Done. Not as attribute. pycbc_live is working fine. Tested OK.